### PR TITLE
Update link to data portal

### PR DESCRIPTION
**Pull request by mgrauer**
keeping up to date with the GUI, which this link will soon point to.

actually, i'm not sure if this is the right way to proceed. i'll ask on another issue.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
   	</div>
   	<div class="collapse navbar-collapse" id="navbar-collapse-1">
   	  <ul class="nav navbar-nav navbar-right">
-  		<li><a href="https://dandiarchive.org/dandiset/drafts">Data Portal</a></li>
+  		<li><a href="https://dandiarchive.org">Data Portal</a></li>
```
Comment (satra) [inline _includes/header.html:15]:
```suggestion
  		<li><a href="https://dandiarchive.org/portal">Data Portal</a></li>
```

Comment (mgrauer) [reply _includes/header.html:15]: I don't follow where this suggestion points to. Are you going to add a new route in the redirector?

**Comment by satra** [comment]
the drafts redirect always takes one to the data portal, and it was temporary to go to the list of datasets quickly. however, i don't think we should point to the primary domain yet. 

i can make another temporary redirect as suggested at `dandiarchive.org/portal`

**Comment by mgrauer** [comment]
If you are suggesting:

- removing drafts redirect
- adding portal redirect to point to gui
- changing this link here to point to portal (as your suggestion)

I'm on board.

**Comment by satra** [comment]
@mgrauer - added https://github.com/dandi/redirector/pull/12

**Pull request merged**